astropy/vo/client/tests/test_conesearch.py

```diff
@@ -149,6 +149,9 @@ def test_async(self):
         async_search = conesearch.AsyncConeSearch(
             SCS_CENTER, SCS_RADIUS, pedantic=self.pedantic)
 
+        # Wait a little for the instance to set up properly
+        time.sleep(3)
```

Comment:
Would it be possible to just try it several times instead of immediatly waiting for 3 seconds.
```
for i in range(3):
   ....
   if not async_search.done():
       time.sleep(1)

assert async_search.done()
```
or is the `async_search.done()` call very time consuming or shouldn't be repeated?